settings/security/third-party/saml.rst

```diff
@@ -19,81 +22,78 @@ Configure SAML in Zammad
+   
+   .. note::
+      - The above certificate is just an example, you should use a proper certificate from your IdP!
+      - Note that this is generally **not** the same cert used by TLS in the HTTPS connection to the IdP.
+
+   .. hint:: **ADFS users**
+      
+      Above certificate refers to the to the signing certificate.
+
+   In Keycloak, this certificate can be found by going to:
+      
+      - Keycloak Admin: Realm settings -> keys -> RSA -> "certifcate" popup
+      - Paste as a single line string with no line breaks.
+      - Do not set this option at all if using the "certificate fingerprint" for cert validation.
+
+IDP certificate fingerprint:
+   Optionally use the fingerprint instead of the certificate itself for cert validation.
+   Because sha1 is used for the fingerprint, this option is not preferred.
+   Do not set this option at all if using the "certificate" field above.
 
 **Name Identifier Format:**
-
-``urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress``
-
-
-Configured example:
-
-.. image:: /images/settings/security/third-party/saml/zammad_connect_saml_thirdparty.png
-   :alt: Zammad SAML Example
-
+   ``urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress``
 
 Configure the SAML IdP
 ----------------------
 
 If your IdP supports xml import or auto xml metadata retrieval, use this URL:
-``https://zammadFQDN/auth/saml/metadata``
+``https://{Zammad-FQDN}/auth/saml/metadata``
 
 For Keycloak IdP, save the contents of that metadata file as "metadata.xml" and then log into Keyclaok and import it as a new Client.
 
 Keycloak sends the "Username" in whatever format your users are stored in. The tested SAML instance used email address for the "username", but Zammad also needs to see an "email" attribute to correctly automap to existing users.
 In Keycloak, we can create a "Mapper" for the newly imported Zammad client.
 
-Goto -> Keycloak Admin -> Clients -> ``https://zammadFQDN/auth/saml/metadata`` -> Mappers -> Create
+Goto -> Keycloak Admin -> Clients -> ``https://{Zammad-FQDN}/auth/saml/metadata`` -> Mappers -> Create
 
 .. code-block:: html
 
    Name: ZammadEmail
-   Mapper Type: User Attribute
+   Mapper Type: User Property
    User Attribute: emailAddress
    SAML Attribute Name: email
    SAML Attribute NameFormat: Basic
 
 Keycloak needs to know the redirect location in advance for security:
-Goto -> Keycloak Admin -> Clients -> ``https://zammadFQDN/auth/saml/metadata``
-
-**Valid Redirect URIs:**
+Goto -> Keycloak Admin -> Clients -> ``https://{Zammad-FQDN}/auth/saml/metadata``
 
-``https://zammadFQDN/auth/saml/callback``
+Valid Redirect URIs:
+   ``https://{Zammad-FQDN}/auth/saml/callback``
 
 For other IdP systems, you can configure things manually.
 Zammad is using POST Bindings for the Assertion Consumer Service (ACS)
 
+ASC POST Binding URL:
+   ``https://{Zammad-FQDN}/auth/saml/callback``
```

Comment:
Same question here. (The XML configuration file at `https://zammad.domain/auth/saml/callback` already includes this URL...)

Reply:
See https://github.com/zammad/zammad-admin-documentation/pull/62#discussion_r476344313 :)